src/Speakable.js

```diff
@@ -1,4 +1,4 @@
-var util = require('util');

```

Comment:
what was the purpose of switching from print to console?

Reply:
print was working and the error was saying that its outdated.